hbt/production/res_networks.py

```diff
@@ -693,10 +658,318 @@ def call_func(self, events: ak.Array, **kwargs) -> ak.Array:
 
         return events
 
+    def teardown_func(self, task: law.Task, **kwargs) -> None:
+        super().teardown_func(task, **kwargs)
+
+        # remove outputs of required fold producers
+        for inp in self.remove_fold_inputs:
+            inp.remove(silent=True)
+
+
+#
+# producers for vbf networks
+#
+
+class _vbf_dnn(_res_dnn_evaluation):
+    """
+    https://gitlab.cern.ch/cclubbtautau/AnalysisCore/-/blob/19cc9e8c02dda68a37c7d9ee308100a9e1821648/src/HHRun3DNNInterface.cc#L325-339
+    """
+
+    uses = {
+        *_res_dnn_evaluation.uses,
+        "Jet.{pt,eta,phi,mass,assignment_bits}",
+        "VBFJet.{pt,eta,phi,mass,btagPNetQvG}",
+        "reg_dnn_moe_nu{1,2}_{px,py,pz}",
+    }
+    require_prodocuers = ["reg_dnn_moe"]
```

Comment:
```suggestion
    require_producers = ["reg_dnn_moe"]
```

Typo only here or is it also wrong in cf?

Reply:
Oh, good catch. That's only a typo here, not in cf.